test/specs/rules/require-anonymous-functions.js

```diff
@@ -30,4 +30,18 @@ describe('rules/require-anonymous-function', function() {
         assert(checker.checkString('$("hi").click(function named(){});').getErrorCount() === 1);
         assert(checker.checkString('var x = function named(){};').getErrorCount() === 1);
     });
+
+    describe('exceptfunctionDeclarations', function() {
```

Comment:
Could you call it `allExcept: ["declarations"]`, we do that in couple places, i guess that would be good trend to follow